docs/source/using-federation/api/apollo-subgraph.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: 'API Reference: @apollo/subgraph'
+api_reference: true
+---
+
+This API reference documents the exports from the `@apollo/subgraph` package. This package enables you to use Apollo Server as a subgraph in a federated supergraph. For more information, see [Implementing a subgraph with Apollo Server](../apollo-subgraph-setup).
```

Comment:
[This article is also pretty much the same](https://www.apollographql.com/docs/federation/api/apollo-subgraph), I mainly updated links, added `MultiCodeBlocks`, and updated references to the `context` object (which I point out in a comment below).